.gitignore

```diff
@@ -8,3 +8,6 @@ stglib.egg-info
 # Sphinx
 doc/_build
 doc/generated
+
+stglib/ea400.py
```

Comment:
This shouldn't be here. I assume this is an old version of the file that got deleted?

Reply:
Deleted line 12 from .gitignore.